src/app/view/components/createLogo.js

```diff
@@ -0,0 +1,17 @@
+import elementCreator from '../../utils/elementCreator';
+import logoPath from '../../../../static/assets/img/logo.png';
+
+const createLogo = () => {
```

Comment:
I think this content is also from another PR. I think it shouldn't be here also

Reply:
deleted